### PR TITLE
Fireworks: Fix in EveService (forward port to master)

### PR DESCRIPTION
**Pull request by alja**
Forward port of #24534

### CODE REVIEW

**Comment by alja** [comment]
+1

File: Fireworks/Eve/plugins/DummyEvelyser.cc [hunk 1]
```diff
@@ -205,41 +208,49 @@ void DummyEvelyser::analyze(const edm::Event& iEvent, const edm::EventSetup& iSe
...
          trk->SetAttLineAttMarker(m_trackList);
          m_trackList->AddElement(trk);
-
+   
          // The display() function runs the GUI event-loop and shows
          // whatever has been registered so far to eve.
          // It returns when user presses the "Step" button (or "Continue" or
          // "Next Event").
-         m_eve->display(std::string("DummyEvelyser::analyze done for:\n") + trk->GetName());
+         //m_eve->display(std::string("DummyEvelyser::analyze done for:\n") + trk->GetName());
```
Comment (fabiocos) [inline Fireworks/Eve/plugins/DummyEvelyser.cc]: @alja for my understanding, what is the implication of this comment? From the comment in the code I would say that a functionality is suppressed, and the loop is just use to update m_tracklist

**Comment by alja** [comment]
@fabiocos 
I have displayed the status on the exit of analyse() function. The EveService::display() call exits DummyEvelyser, so it should not be in the construction of the graphic scene.

**Comment by alja** [comment]
@rovere , here are brief instructions
https://twiki.cern.ch/twiki/bin/view/CMS/PhysicsToolsDevFireworksFFWEveService

**Comment by alja** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**